linker/jvm/src/main/scala/org/scalajs/linker/backend/closure/ClosureModuleBuilder.scala

```diff
@@ -32,8 +33,17 @@ private[closure] class ClosureModuleBuilder(
   private val treeBuf = mutable.ListBuffer.empty[Node]
   private val module = new JSModule("Scala.js")
 
-  def addJSTree(tree: Tree): Unit =
-    treeBuf += transformer.transformStat(tree)(NoPosition)
+  def addJSTree(tree: Tree): Unit = {
+    tree match {
+      case js.Block(stats) =>
+        for (stat <- transformer.transformBlockStats(stats)(NoPosition))
+          treeBuf += stat
+      case js.Skip() =>
+        // ignore
+      case _ =>
+        treeBuf += transformer.transformStat(tree)(NoPosition)
+    }
+  }
```

Comment:
This is odd: Why is this necessary now? In other words: We might have an incomplete specification of `addJSTree` or an abuse of a `js.Block` instead of calling `addJSTree` multiple times.

Reply:
I see; I think this warrants a comment. Also:

```scala
treeBuf ++= transformer.transformBlockStats(stats)(NoPosition)
```

in the `js.Block` case?